plugins/dependency-checker/src/main/java/com/freenow/sauron/plugins/generator/python/PythonDependencyGenerator.java

```diff
@@ -49,36 +44,26 @@ public Path generateCycloneDxBom(Path repositoryPath)
     {
         try
         {
-            createPythonVirtualEnv(repositoryPath);
             generateRequirementsFreeze(repositoryPath);
-
-            return repositoryPath.resolve("bom.xml");
+            Path resolved = repositoryPath.resolve("bom.xml");
+            if (resolved.toFile().exists() && Files.size(resolved) > 0)
+            {
+                log.info("BOM file created.");
+            }
+            else
+            {
+                log.info("BOM file is either empty or does not exist.");
```

Comment:
Can you log the result of the resolved.toFile().exists() and Files.size(resolved)

Reply:
we keep as is as at this phase the file does not exists or is empty.